c2corg_common/fields_book.py

```diff
@@ -27,6 +28,7 @@
     'author',
     'quality',
     'publication_date',
+    'langs',
```

Comment:
Do we really need all those attributes in the cards?
I am pretty sure we don't need the ``description`` for instance.

I would say (it's better to have too few than too much):
```
LISTING_FIELDS = [
     'locales.title',
     'activities',
￼     'author',
￼     'quality'
￼  ]
```